--- a/massive-passive-client
+++ b/massive-passive-client
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/massive-passive-server
+++ b/massive-passive-server
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
@@ -156,4 +156,4 @@
     main.start()
     sys.exit(0)
 
-# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
\ No newline at end of file
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
--- a/mplib/__init__.py
+++ b/mplib/__init__.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/mplib/cmd.py
+++ b/mplib/cmd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/mplib/config.py
+++ b/mplib/config.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/mplib/daemon.py
+++ b/mplib/daemon.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/mplib/getopts.py
+++ b/mplib/getopts.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/mplib/log.py
+++ b/mplib/log.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/mplib/net/__init__.py
+++ b/mplib/net/__init__.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/mplib/net/client.py
+++ b/mplib/net/client.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/mplib/net/server.py
+++ b/mplib/net/server.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/mplib/scheduler.py
+++ b/mplib/scheduler.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
--- a/mplib/threads.py
+++ b/mplib/threads.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 
 __author__ = 'Robin Wittler'
