Index: massive-passive_0.2.11/etc/default/massive-passive
===================================================================
--- massive-passive_0.2.11.orig/etc/default/massive-passive	2013-01-15 19:01:35.000000000 +0100
+++ massive-passive_0.2.11/etc/default/massive-passive	2013-01-15 19:09:49.763205316 +0100
@@ -7,14 +7,14 @@
 #
 
 LOGLEVEL="info"
-PIDFILE="/tmp/massive-passive.pid"
+PIDFILE="/var/log/massive-passive/massive-passive.pid"
 CONFDIR="/etc/massive-passive/checks.d"
-USER="nobody"
-GROUP="nogroup"
+USER="nagios"
+GROUP="nagios"
 BATCH_WAIT_TIME="2"
 BATCH_MAX_ITEMS="10"
 INITIAL_RANDOM_WAIT_RANGE="10"
-LOGFILE="/tmp/massive-passive.log"
+LOGFILE="/var/log/massive-passive/massive-passive.log"
 PATH_TO_SEND_NSCA="/usr/sbin/send_nsca"
 
 
