Index: massive-passive/etc/default/massive-passive
===================================================================
--- massive-passive.orig/etc/default/massive-passive	2012-12-28 13:26:13.000000000 +0100
+++ massive-passive/etc/default/massive-passive	2013-01-02 13:14:24.911437427 +0100
@@ -8,4 +8,4 @@
 PIDFILE="/tmp/massive-passive.pid"
 
 # Additional options that are passed to the Daemon.
-DAEMON_ARGS="--loglevel info --user nobody --group nogroup --batch-mode --batch-wait-time 2 --pidfile $PIDFILE"
+DAEMON_ARGS="--loglevel info --user nagios --group nagios --batch-mode --batch-wait-time 2 --pidfile $PIDFILE"
